smart-contracts/contracts/Unlock.sol

```diff
@@ -98,6 +103,9 @@ contract Unlock is UnlockInitializable, UnlockOwnable {
   mapping(uint16 => address) private _publicLockImpls;
   uint16 public publicLockLatestVersion;
 
+  // Uniswap
+  address public uniswapRouter;
```

Comment:
Can we actually pass that as a part of the `swapAndCall` as this would let us be more flexible?